data/pv-forecast.json

```diff
@@ -1,259 +1,259 @@
 {
   "forecast_uuid": "95833950-de0f-4447-8c73-ffa5138b225d",
   "site_uuid": "b97f68cd-50e0-49bb-a850-108d4a9f7b7e",
-  "forecast_creation_datetime": "2023-04-16T01:26:20.557211+00:00",
+  "forecast_creation_datetime": "2024-04-16T01:26:20.557211+00:00",
```

Comment:
Is there any reason for bumping these years up?

Reply:
So the Sunny Timeframe module displays a "Sunny in..." message.

Reply:
Gotcha